### PR TITLE
feat: add conditional for local server endpoint

### PR DESCRIPTION
**Pull request by theyokohamalife**
## 🔧 What changed
This adds a new command to the package.json that can be used to connect the FE to findadoc-server when a dev is running it locally.

## 🧪 Testing instructions
1. Follow the steps in the findadoc-server repo to start the Firebase database and backend server locally.
2. Start the frontend server using the `yarn dev:localdb` command.
3. Confirm that the FE displays data from the local database.

### CODE REVIEW

**Review by NabbeunNabi — CHANGES_REQUESTED** [review]
I am following the backend readme and there are some issues there, which might lead to others not being able to use this command.

### Screenshots with Comments

- This command said it was not found
image

- For PC this might be the chocolately command now to install java? [Chocolatey Java](https://community.chocolatey.org/packages/javaruntime )
-Current instructions for Java
image

-Got stuck started the firebase because I am not sure what to do here since the up and down arrows  did nothing

image

So might need to fix the backend read me before merging this to make sure everyone can set it up to work on it properly before running this command

**Review by NabbeunNabi — APPROVED** [review]

**Comment by NabbeunNabi** [comment]
I updated the readME by going through step by step. So this can be merged as long as this PR https://github.com/ourjapanlife/findadoc-server/pull/454 is merged

File: package.json
Comment (ermish) [inline package.json:8]: localdb doesn't make sense here as we're not starting a database. 
I would just add the NODE_ENV variable to the `dev` task

File: utils/graphql.ts
Comment (ermish) [inline utils/graphql.ts:10]:
```suggestion
    let apiURL;
```

File: utils/graphql.ts
Comment (ermish) [inline utils/graphql.ts:13]:
```suggestion
        apiURL = "http://127.0.0.1:4000"
```

File: utils/graphql.ts
Comment (ermish) [inline utils/graphql.ts:16]:
```suggestion
        apiURL = "https://api.findadoc.jp"
```

File: README.md
Comment (ermish) [inline README.md:98]: would remove this. see next comment

**Review by ermish — CHANGES_REQUESTED** [review]
reviewed!

**Review by ermish — APPROVED** [review]
everything looks good except the addition of a bunch of semicolons 🤔 do you have pretter installed or something?

File: utils/graphql.ts
Comment (NabbeunNabi) [reply utils/graphql.ts:7]: I use the ESLINT that you told me to use. `.prettierignore` is still there. I just changed the variable names. The semicolons were already there so I thought there might be a reason so I didn't remove them here.

Comment (NabbeunNabi) [reply utils/graphql.ts]: All gone. Can you check again @ermish

**Review by ermish — APPROVED** [review]
Sheep it 🐑

**Pull request merged**
